services/gas-api/quickstart.md

```diff
@@ -45,18 +72,23 @@ npm install dotenv
 
 Create a `.env` file at the project root and add the following data:
 
+:::caution warning
+Do not commit the `.env` file to your repository if it contains sensitive data.
+You can [create a `.gitignore`
+file](https://docs.infura.io/tutorials/developer-tools/javascript-dotenv#create-a-.gitignore-file)
+to prevent accidentally committing the file.
+:::
+
 ```bash title=".env"
 INFURA_API_KEY=<API-KEY>
 INFURA_API_KEY_SECRET=<API-KEY-SECRET>
 ```
 
 Replace the Infura project credential placeholders with your own.
 
-:::caution warning
-Do not commit the `.env` file to your repository if it contains sensitive data.
-You can [create a `.gitignore`
-file](https://docs.infura.io/tutorials/developer-tools/javascript-dotenv#create-a-.gitignore-file)
-to prevent accidentally committing the file.
+:::note
+The `INFURA_API_KEY_SECRET` is optional and only necessary if you are using an
+[API key secret](https://docs.infura.io/dashboard/secure-an-api/api-key-secret) to authenticate requests.
 :::
 
 ## Create your script
```

Comment:
Can you make changes in this section as in the Infura PR? https://github.com/INFURA/docs/pull/234/files#diff-48d2028b36bfd76af81aba7ebdb0d5dea924cf5d9fbd713b91cea4944a78f85cR71